config/propelOrmPluginConfiguration.class.php

```diff
@@ -27,7 +29,7 @@ public function initialize()
     {
       if (sfConfig::get('sf_debug') && sfConfig::get('sf_logging_enabled'))
       {
-        Propel::setLogger(new sfPropelLogger($this->dispatcher));
+        Propel::setLogger(Service::get(LoggerInterface::class));
```

Comment:
This is not great, as `Service::get()` is our app extension and is not a part of the framework. Just a note,  we can address this some day later.

Reply:
I agree, and was aware that the solution is not perfect, but I went with the quick/easy approach just to avoid spending too much time on not so important parts, since the upgrade was a huge take.

But thanks for the remark!